src/routes/login/components/LoginPage.js

```diff
@@ -0,0 +1,121 @@
+const React = require('react');
+const T = require('prop-types');
+const { NavLink } = require('react-router-dom');
+const { default: Styled } = require('styled-components');
+const { default: Typography } = require('@material-ui/core/Typography');
+const { default: TextField } = require('@material-ui/core/TextField');
+const { default: Button } = require('@material-ui/core/Button');
+const { default: Box } = require('@material-ui/core/Box');
+const StrangeForms = require('strange-forms');
+
+const internals = {};
+
+module.exports = class LoginPage extends StrangeForms(React.Component) {
+
+    static propTypes = {
+        reqCreateAccount: T.func.isRequired,
+        isAuthenticated: T.bool
+    };
+
+    static fields = {
+        email: '',
+        password: ''
+    };
+
+    constructor(props) {
+
+        super(props);
+
+        this.state = {
+            email: '',
+            password: '',
+            isSubmitting: false
+        };
+
+        this.strangeForm({
+            fields: Object.keys(LoginPage.fields),
+            get: (someProps, field) => {
```

Comment:
This looks like it's trying to nab default/controlled values off of a prop named `data`.

Reply:
Good point; I'll ditch that unnecessary code and just have this return `''`.